src/components/blockSettings/BlockAlways.tsx

```diff
@@ -24,13 +27,14 @@ export default function BlockAlways() {
         <p className={`text-sm font-bold  ${isScheduled ? 'text-primary': 'text-gray-500'}`}>Schedule</p>
       </div>
       <Button
-        aria-label="activate"
+        aria-label="activate-extension"
         type="button"
         variant="default"
         size="default"
         className="px-10"
+        onClick={handleActivateExtension}
```

Comment:
Hey @Riri202, `toggleExtensionState` could be a better name for `handleActivateExtension` function since it handles both activation and deactivation